tests/pinquerycontext.pinmap

```diff
@@ -0,0 +1,46 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<PinMap schemaVersion="1.1" xmlns="http://www.ni.com/TestStand/SemiconductorModule/PinMap.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<Instruments>
+		<Instrument name="CustomInstrument1" instrumentTypeId="CustomInstrumentTypeId1">
```

Comment:
While it is fine to use custom instruments for this test, it might be simpler to just use a couple of NI-Digital instruments and avoid all of these channel group definitions which are not really necessary.